ios/Runner/AppDelegate.swift

```diff
@@ -16,6 +17,9 @@ import Firebase
     }
     GMSServices.provideAPIKey("AIzaSyC5szIWBSfTg3SzJPkTPU7DPfZcdkvFd4A")
     GeneratedPluginRegistrant.register(with: self)
+    UIApplication.shared.setMinimumBackgroundFetchInterval(TimeInterval(60*15))
+    WorkmanagerPlugin.registerBGProcessingTask(withIdentifier: "trackingTask")
+    WorkmanagerPlugin.registerBGProcessingTask(withIdentifier: "scheduleDailyTasks")
```

Comment:
Not sure but... Shouldn't this be
`WorkmanagerPlugin.registerPeriodicTask(withIdentifier: "scheduleDailyTasks", frequency: NSNumber(value: 24 * 60 * 60))
`